src/Vagrantfile

```diff
@@ -66,16 +71,17 @@ Vagrant.configure("2") do |config|
      dpkg-reconfigure -p medium lxd -fnoninteractive
   SHELL
 
-  config.vm.provision "apt", type: "shell", inline: <<-SHELL
+  config.vm.provision 'config', type: 'shell', inline: <<-SHELL
      apt-add-repository ppa:ansible/ansible
-     apt-get update
-     apt-get install -y git gitk git-gui kdiff3 software-properties-common ansible python-pip
-     pip install docker-py
+     apt-get -qqy update
+     apt-get -qqy install virtualbox-guest-utils linux-virtual lxd git software-properties-common ansible python-docker
+     adduser vagrant lxd
      lxd init --auto --storage-backend=dir
      echo net.ipv4.ip_forward=1 >>/etc/sysctl.conf
      sysctl -p
+     sudo sed -i'' -r 's/(AcceptEnv*)/#\\1/' /etc/ssh/sshd_config
+     systemctl reload sshd
   SHELL
 
-  config.vm.provision "prepare_buildhost", type: "shell", path: "prepare_buildhost.sh", privileged: false
-
+  config.vm.provision 'prepare_buildhost', type: 'shell', path: 'prepare_buildhost.sh', privileged: false
```

Comment:
Just wondering if `prepare_buildhost.sh` should be called here. If it should, why not call `update_ruby_dependencies.sh` also? The other option would be to leave this to the user and just follow the build instructions.

Reply:
No strong opinion here, could be removed or kept as is.